### PR TITLE
Release/v0.4.4: Style guidelines and Documentation

### PR DESCRIPTION
**Pull request by matbmoser**
# Release v0.4.4
## Updated
- Reworked header component and styles to match portal
- Reworked footer component and styles to match portal
- Reworked search input and styles to match portal
- Updated image repository for consumer-ui and consumer-backend helm deployments
- Reworked table component and styles to match portal

## Added
- Refactored SCSS styles structure, allowing a optimized scalability from the application
- Updated frontend dependencies file.
- Matched portal styles to pass quality gate 5
- Refactored responsiveness from components
- Important Documentation related with the product and repository

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2023 BASF SE, BMW AG, Henkel AG & Co. KGaA
```
Comment (matbmoser) [inline jsconfig.json:1]: JSONS dont have copyright header

File: README.md [hunk 2]
```diff
@@ -14,22 +14,35 @@
  limitations under the License.
 -->
 
-# ![Product Battery Passport Consumer Application (Frontend)](./docs/catena-x-logo.svg) Product Battery Passport Consumer App (Frontend)
```
Comment (matbmoser) [inline README.md:17]: Refactored the Readme

File: src/assets/plugins/webfontloader.js [hunk 3]
```diff
@@ -25,7 +25,11 @@ export async function loadFonts () {
 
   webFontLoader.load({
     google: {
-      families: ['Roboto:100,300,400,500,700,900&display=swap'],
+      families: [
```
Comment (matbmoser) [inline src/assets/plugins/webfontloader.js:28]: fonts for matching portal

File: src/assets/styles/components/general/footer.scss [hunk 4]
```diff
@@ -0,0 +1,67 @@
+/**
```
Comment (matbmoser) [inline src/assets/styles/components/general/footer.scss:1]: Styles refactored to pass styleguidelines

File: src/components/general/Footer.vue [hunk 5]
```diff
@@ -1,110 +1,106 @@
...
-@media(max-width: 740px){
-  
-.links {
-  padding-top: 20px;
-  display: initial;
-}
-}
-
-</style>
+<!--
```
Comment (matbmoser) [inline src/components/general/Footer.vue:1]: Components abstracted in refactor

File: src/components/passport/sections/CellChemistry.vue [hunk 6]
```diff
@@ -1,167 +1,169 @@
...
+    <SectionHeader title="2. Cell chemistry" @click="toggle = !toggle" />
+    <div class="section-content" :class="[toggle ? 'hidden' : '']">
+      <!-- Composition of Electrolyte -->
+      <AttributeField
+        :attributes-list="cellChemistry.electrolyteComposition"
+        label="Composition of electrolyte"
+        data-cy="electrolyte-composition"
+      />
+      <!-- Composition of Anode -->
+      <AttributeField
```
Comment (matbmoser) [inline src/components/passport/sections/CellChemistry.vue:28]: Simplified the component with abstraction removing duplication

File: src/views/HomeView.vue [hunk 7]
```diff
@@ -43,7 +51,7 @@
 </template>
 
 <script>
-import QRScannerView from "./SearchView.vue";
+import SearchView from "./SearchView.vue";
```
Comment (matbmoser) [inline src/views/HomeView.vue:54]: Renamed components to ease the code understading

File: src/views/SearchView.vue [hunk 8]
```diff
@@ -177,6 +178,9 @@ p {
 }
 
 @media (max-width: 856px) {
+  .qr-container {
```
Comment (matbmoser) [inline src/views/SearchView.vue:181]: Fixed bugs releated to style guidelines

**Comment by matbmoser** [comment]
@SebastianBezold please review the pull request, I added some comments to ease your understanding

**Comment by SebastianBezold** [comment]
Hi @matbmoser,
the changes look good to me, but i cannot judge, if the styleguide is now actually met, or if your arc42 docs are correct. 
Can you please ask someone, who can do the content review for this PR like mentioned on our [Developer Hub](https://eclipse-tractusx.github.io/docs/oss/code-reviews).
I already looked at the changes to the workflows, the Chart and the legal documentation. They all seem fine

**Comment by jsmunich** [comment]
Style Guideline was approved within: https://jira.catena-x.net/browse/CMP-435
Documentation was approved contenwise within QG4: https://confluence.catena-x.net/display/CDPP/New%3A+Consortia+QG+4+FOSS+R3.0.0
This is only the move from confluence to GitHub.

**Comment by thorstendikmann** [comment]
Documentation and Styleguide changes were reviewed. They mach the Release 3 requirements.

**Comment by SebastianBezold** [comment]
Thanks for the additional review @thorstendikmann!

**Pull request merged**